configure

```diff
@@ -133,6 +133,20 @@ else
 fi
 rm -Rf pctest*
 
+# Provide macros to avoid calling 'buffer->GetBackingStore()->Data()' due to
+# a UBSAN warning: https://github.com/jeroen/V8/issues/152
+if [ "$IS_MACOS" ] || [ "$IS_UBUNTU" ] || [ "$IS_RHEL" ] || [ "$CI" ]; then
```

Comment:
Should check: !static_download. Otherwise there's a chance that we end on a false back port.